website/docs/r/service.html.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: "secrethub"
+page_title: "Resource: secrethub_service"
+sidebar_current: "docs-secrethub-resource-service"
```

Comment:
Don't forget to add this to https://github.com/secrethub/terraform-provider-secrethub/blob/feature/resource-service/website/secrethub.erb as well.